### ISSUE TITLE
scripts/puppeteer.js is missing in nightly

### ISSUE THREAD

**Issue by dan-m-joh** [sni/Thruk#1061]
Hello Sven,

I downloaded the nightly build of Thruk (2.39-20201124.rhel8) to try the new puppeteer grafana rendering.

There were some issues with this build.

1. scripts/puppeteer.js is missing in the rpm's (I had to download this manually)
2. the grafana_export.sh script did not work for me unless I changed the following:
```
@@ -22,10 +22,10 @@

 INPUT="$INPUT&from=$START&to=$END"

-if [ -n $OMD_ROOT ]; then
+if [ -n "$OMD_ROOT" ]; then
     export NODE_PATH=$OMD_ROOT/node_modules/
 fi
-if [ -n $NODE_PATH ] && [ -d $NODE_PATH ]; then
+if [ -n "$NODE_PATH" ] && [ -d "$NODE_PATH" ]; then
     node $DIR/puppeteer.js "$INPUT" "${TEMPFILE}.png" "$WIDTH" "$HEIGHT" "$THRUK_SESSION_ID" 2>&1
     rc=$?
 else
```
Regards,

Dan

**Comment by sni** [comment]
thanks for testing, should be better tomorrow. Besides that, is it working?

**Comment by dan-m-joh** [comment]
Yes, (after "fixing" the two issues above).

And, of cause, setting `NODE_PATH` and in my case `PUPPETEER_EXECUTABLE_PATH` in `~apache/.thruk`

The reason for setting `PUPPETEER_EXECUTABLE_PATH` is that `npm install puppeteer` could (for some reason) not download and install the "bundled" Chromium so I had to install Chromium separately.

For reference, I had to install puppeteer like this (as root):
```
export PUPPETEER_SKIP_DOWNLOAD=1
npm install -g puppeteer
```
Thank you very much looking into this.

Regards,
Dan
